scripts/store/explore.js

```diff
@@ -138,6 +140,64 @@ function GenerateQueue()
 		} );
 }
 
+function ClaimSaleItem()
+{
+	const applicationConfigElement = document.getElementById( 'application_config' );
+	if( applicationConfigElement )
+	{
+		const userConfigJSON = applicationConfigElement.dataset.store_user_config;
+		const webapiToken =
+			userConfigJSON && JSON.parse( userConfigJSON ).webapi_token;
+
+		if( webapiToken )
+		{
+			let result = span.textContent; // storing explore result to show again after trying to get a sale item
+			span.textContent = _t( 'explore_saleitem_trying_to_claim' );
+
+			fetch(
+				`https://api.steampowered.com/ISaleItemRewardsService/ClaimItem/v1?access_token=${webapiToken}`,
```

Comment:
Use urlsearchparams, will guarantee to work if they happen to require encoding or something